dedupe/deduper.py

```diff
@@ -33,8 +33,7 @@ def __init__(self,
                  duplicate_package_log=None,
                  collection_package_log=None,
                  run_id=None,
-                 oldest=True,
-                 update_name=False):
+                 oldest=True):
```

Comment:
All these changes are from changes to main, you should rebase.
The additions in main are similar to the ones you need to make here;
add an optional parameter to take the `identifier_type` that is then passed to the ckan_api functions where needed. This will be either `identifier` or `guid`, defaulting to `identifier`.
In the [duplicates-identifier-api.py](https://github.com/GSA/datagov-dedupe/blob/main/duplicates-identifier-api.py#L56-L57) script, you'll need to add a similar option to accept a `identifier-type` parameter that can accept a different field, in this case `guid`.